flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServicesConfiguration.java

```diff
@@ -384,13 +383,6 @@ private static NetworkEnvironmentConfiguration parseNetworkEnvironmentConfigurat
 			ConfigConstants.TASK_MANAGER_NETWORK_DEFAULT_IO_MODE,
```

Comment:
I guess we can also delete the option `syncOrAsync` and `TASK_MANAGER_NETWORK_DEFAULT_IO_MODE` and `IOManager.IOMode` enum.

Reply:
Yes, I missed that part.